spec/controllers/shipments_controller_spec.rb

```diff
@@ -0,0 +1,113 @@
+require 'rails_helper'
+require 'support/vcr_setup'
+
+RSpec.describe ShipmentsController, type: :controller do
+  describe "GET #index" do
+    it "responds with an error and status code 404" do
+      get :index
+      expect(response.response_code).to eq 404
+      expect(eval(response.body)[:errors]).to include("This page does not exist. Perhaps you were looking for 'adaships.herokuapp.com/log' or 'adaships.herokuapp.com/ship'.")
```

Comment:
I think you want `JSON.parse` here instead of `eval` right? The `eval` run with the contents of a web request make me nervous from a security perspective. Even if it's test code and your own controller.